app/app/utils.py

```diff
@@ -48,32 +48,45 @@ def add_contributors(repo_data):
     return repo_data
 
 
-def sync_profile(handle):
-    data = get_user(handle)
-    is_error = 'name' not in data.keys()
-    if is_error:
-        print("- error main")
-        rollbar.report_message('Failed to fetch github username', 'warning', extra_data=data)
-        return None
-
-    repos_data = get_user(handle, '/repos')
-    repos_data = sorted(repos_data, key=lambda repo: repo['stargazers_count'], reverse=True)
-    repos_data = [add_contributors(repo_data) for repo_data in repos_data]
+def get_profile(handle, sync=True):
```

Comment:
i forgot that the `sync_profiles` management command already takes care of syncing (and does it out the context of a request... which is great.

i think this PR would be massively simplified if i were to remove all the changes related to sync_profile/get_profile and just add the `record_user_action` stuff.  planning on doing that soon